.github/workflows/deploy.yml

```diff
@@ -30,8 +30,7 @@ jobs:
         make build
     - name: Post to Splunk SOAR API
       run: |
-        # TODO: (#8) Re-enable deployment - waiting on SOAR API key
-        # make deploy
+        make deploy
       env:
```

Comment:
I think the error indicates that an item is missing here.

Reply:
Yep. Added.